### PR TITLE
privacy issues and script cleanup

### PR DESCRIPTION
**Pull request by ethn**
I'm happy to restore any script here with code that you think will be useful as a reference to us in the future, but I wanted to clean up a bit.

We could also consider moving them to an "old" directory...

### CODE REVIEW

File: mod/wikirate/set/all/wikirate/google_analytics.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 format :html do
   def google_analytics_snippet_vars
-    super.insert 1, [:_setPageGroup, 1, card.type_name]
+    super.insert(1, [:_setPageGroup, 1, card.type_name]).insert 2, ['_gat._anonymizeIp']
```
Comment (ethn) [inline mod/wikirate/set/all/wikirate/google_analytics.rb]: IP anonymization

File: script/cron/delete_old_ip_addresses.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env ruby
+
+require File.expand_path("../../../config/environment", __FILE__)
+
+Card::Act.where("acted_at < ?", 6.months.ago).update_all ip_address: nil
```
Comment (ethn) [inline script/cron/delete_old_ip_addresses.rb]: deleting old IP addresses.

File: script/cron/table_maintenance.rb [hunk 3]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env ruby
 
-require File.dirname(__FILE__) + "/../config/environment"
+require File.expand_path("../../../config/environment", __FILE__)
```
Comment (ethn) [inline script/cron/table_maintenance.rb:3]: note: will need to update cron jobs upon deploy!

File: VERSION [hunk 4]
```diff
@@ -1 +1 @@
-0.15.0
```
Comment (ethn) [inline VERSION:1]: wait, we never deployed 0.15.0, so this tick shouldn't happen yet.  will fix

**Comment by xithan** [comment]
I don't see any value in removing all the scripts. I would keep them.

**Review by xithan — APPROVED** [review]

**Pull request merged**